script/testing/junit/run_junit.py

```diff
@@ -29,15 +43,43 @@
                          type=int,
                          help="Threshold under the 'extened' query mode")
 
+    download_git_repo()
     args = vars(aparser.parse_args())
-
+    exit_code = 0
+    code = []
+    # FIXME: Extended protocol is broken for some queries, so we are
+    # overriding the query mode to always be simple
+    args['query_mode'] = 'simple'
```

Comment:
I changed my mind. Let's just remove this code and then we'll modify the Jenksfile